src/utils/server-utils.ts

```diff
@@ -154,32 +153,23 @@ async function getWCSCoverage(serverUri: string) {
       'domainSet.temporalDomain.gml:timePosition',
     );
   } catch (error) {
-    // TODO we used to throw the error here so a notification appears. Removed because a failure of one shouldn't prevent the successful requests from saving.
+    // TODO we used to throw the error here so a notification appears via middleware. Removed because a failure of one shouldn't prevent the successful requests from saving.
```

Comment:
I'd rather open a small `enhancement` issue and reference it in here to add more context rather than detail everything.

TODO - send a notification on error. See issue #XXX for more information.

Reply:
Done: #114